pkg/kudoctl/cmd/install/install.go

```diff
@@ -213,60 +214,43 @@ func verifySingleFramework(name, previous string, gc *github.Client, k2c *kudo.C
 			if helpers.AskForConfirmation() {
 				// If Instance is a dependency we need to make sure installSingleInstanceToCluster is aware of it.
 				// By having the previous string set we can make this distinction.
-				err := installSingleInstanceToCluster(name, previous, *content.Path, gc, k2c)
-				if err != nil {
+				if err := installSingleInstanceToCluster(name, previous, bundle.Instance, kc); err != nil {
 					return errors.Wrap(err, "installing single Instance")
 				}
 			}
 		} else {
-			err := installSingleInstanceToCluster(name, previous, *content.Path, gc, k2c)
-			if err != nil {
+			if err := installSingleInstanceToCluster(name, previous, bundle.Instance, kc); err != nil {
 				return errors.Wrap(err, "installing single Instance")
 			}
 		}
 
 	}
-
 	return nil
 }
 
 // Todo: needs testing
 // installSingleFrameworkToCluster installs a given Framework to the cluster
-func installSingleFrameworkToCluster(name, path string, gc *github.Client, k2c *kudo.Client) error {
-	frameworkYaml, err := gc.GetFrameworkYaml(name, path)
-	if err != nil {
-		return errors.Wrapf(err, "getting %s-framework.yaml", name)
-	}
-	_, err = k2c.InstallFrameworkObjToCluster(frameworkYaml)
-	if err != nil {
+func installSingleFrameworkToCluster(name string, f *v1alpha1.Framework, kc *kudo.Client) error {
+	if _, err := kc.InstallFrameworkObjToCluster(f); err != nil {
 		return errors.Wrapf(err, "installing %s-framework.yaml", name)
 	}
-	fmt.Printf("framework.%s/%s created\n", frameworkYaml.APIVersion, frameworkYaml.Name)
+	log.Infof("framework.%s/%s created\n", f.APIVersion, f.Name)
```

Comment:
When installing a framework it would look now like this e.g.:

```
$ go run cmd/kubectl-kudo/main.go install kafka --all-dependenciesNo Instance tied to this "zookeeper" version has been found. Do you want to create one? (Yes/no) 
No Instance tied to this "kafka" version has been found. Do you want to create one? (Yes/no) 
```

instead of the kubectl used style with 

```
framework.kudo.k8s.io "kafka" created
frameworkversion.kudo.k8s.io "kafka-0.1.0" created
instance.kudo.k8s.io "kafka" created
```
I just noticed and am indifferent here, probably the output should anyways be an IOStreams we add later on and this could pave the way. Wdyt?

Reply:
Also because when framework and frameworkversion is already installed we print out:

```
framework.kudo.k8s.io/kafka unchanged
frameworkversion.kudo.k8s.io/kafka unchanged
framework.kudo.k8s.io/zookeeper unchanged
frameworkversion.kudo.k8s.io/zookeeper unchanged
```

that should then maybe be not printed out as well.